### PR TITLE
Fix issue in source code view when there is no samples

### PR DESCRIPTION
**Pull request by karupayun**
Show 0.00 % if there isn't any sample.

### CODE REVIEW

**Comment by beckerhe** [comment]
I think we misunderstood each other. If there is no data available (the returned optional  has no value, then we want to show nothing.) we only want to show a number if we have data available.

File: src/CodeViewer/Viewer.cpp [hunk 1]
```diff
@@ -239,9 +239,13 @@ void Viewer::DrawSampleCounters(QPaintEvent* event) {
                                fontMetrics().height()};
       painter.setPen(kLineNumberForegroundColor);
 
+      double function_percentage =
+          (code_report_->GetNumSamplesInFunction()
+               ? static_cast<double>(num_samples_in_line) / code_report_->GetNumSamplesInFunction()
+               : 0.);
```
Comment (beckerhe) [inline src/CodeViewer/Viewer.cpp:245]: Could you make a small static function out of this?

**Review by beckerhe — APPROVED** [review]
Otherwise LGTM

File: src/CodeViewer/Viewer.cpp (hunk 1 above)
Comment (karupayun) [reply src/CodeViewer/Viewer.cpp]: I modified DoubleToPercentageString for FractionToPercentageString. Seems to be easier to read.

Comment (beckerhe) [reply src/CodeViewer/Viewer.cpp]: Even better.

**Pull request merged**